src/Compilers/Core/Portable/PEWriter/PeWriter.cs

```diff
@@ -318,22 +333,33 @@ protected override void Serialize(BlobBuilder builder, SectionLocation location)
             }
         }
 
-        private static int CalculateStrongNameSignatureSize(CommonPEModuleBuilder module)
+        private static int CalculateStrongNameSignatureSize(CommonPEModuleBuilder module, RSAParameters? privateKey)
```

Comment:
>CalculateStrongNameSignatureSize [](start = 27, length = 32)

I suggest moving this to utility as well.

Reply:
Done.